.github/workflows/smoke_tests.yaml

```diff
@@ -0,0 +1,33 @@
+# only has to pass for python 3.9
+name: Smoke Tests
+
+on:
+  push:
+    branches:
+      main
```

Comment:
Just want to verify my understanding, does this mean that these tests would run on any forced pushes to main? Just want to make sure we don't run them each time we push to origin on development branches

Reply:
Not sure if I understand your concern. Any pushes to main will trigger the workflow, both forced or via pull request merge. Let me know if you see any problems with that.
I assume by development branches you mean branches like this one, right? On the config below the line you commented, it establishes that it will also run when a pull request targeting main is updated (like this one) from any branch. It will not run if there is an update to a branch that doesn't have a pull request into main opened for it.

I copied this from the `tests.yaml` file, so whenever the tests run this will also run.